### DISCUSSION TITLE
RFI

### DISCUSSION THREAD

**Discussion by kcabral817** [MarimerLLC/csla#3246] [Questions]
I hope it's okay to ask.  I am looking for a Saas hosting company that has IIS and Sql Server support.  My CSLA ERP application is almost ready for prime time and I am asking if anyone knows of a company?  I checked MS Asure but, OMG, there pricing would leave me homeless.  I've looked around on the web, but wanted to get input from you folks.

**Comment by rockfordlhotka** [comment]
Just trying to be productive: Isn't the idea of SaaS such that you price in the cost of hosting?

In other words, if you use Azure or AWS, and there's a $x per hour cost to run an IIS host (possibly a VM?), then you pass that cost along to your customers.
